static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -150,6 +150,21 @@ define([
         });
     }
 
+    function initHeroic(){
+      var player = $('.vjs-tech')[0];
```

Comment:
might be worth wrapping this in a try/catch or checking that each of these elements exists